packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -77,10 +86,30 @@ export class FluidCache implements IPersistedCache {
 	private dbReuseCount: number = -1;
 
 	constructor(config: FluidCacheConfig) {
-		this.logger = createChildLogger({ logger: config.logger });
-		this.partitionKey = config.partitionKey;
-		this.maxCacheItemAge = config.maxCacheItemAge;
-		this.closeDbAfterMs = config.closeDbAfterMs ?? 0;
+		const { logger, partitionKey, maxCacheItemAge, closeDbAfterMs } = config;
+		this.logger = createChildLogger({ logger });
+		this.partitionKey = partitionKey;
+		if (maxCacheItemAge > maxCacheExpiryTimeoutMs) {
+			const error = new UsageError(
+				`maxCacheItemAge(${maxCacheItemAge}) cannot be greater than ${maxCacheExpiryTimeoutMs}`,
+				{
+					maxCacheItemAge,
+					maxCacheExpiryTimeoutMs,
+					pkgVersion,
+				},
+			);
+			// go with logging, rather than throwing for now
+			// as throwing could break existing usages
+			this.logger.sendErrorEvent(
+				{
+					eventName: "maxCacheItemAgeTooLarge",
+					subCategory: FluidCacheEventSubCategories.FluidCache,
+				},
+				error,
+			);
+		}
+		this.maxCacheItemAge = maxCacheItemAge;
```

Comment:
Might as well do math.min here?  I guess it doesn't matter either way, but would be more explicit if you read this code (which a consumer is more likely to do than the epochTracker code that enforces the universal max)